core/rawdb/freezer_batch.go

```diff
@@ -19,28 +19,30 @@ package rawdb
 import (
 	"bytes"
 	"fmt"
+	"sync"
+	"sync/atomic"
 
 	"github.com/ethereum/go-ethereum/common/math"
 	"github.com/ethereum/go-ethereum/rlp"
-	"sync/atomic"
 )
 
-// freezerBatch is a batch for a freezer table.
-type freezerBatch struct {
+// freezerTableBatch is a batch for a freezer table.
+type freezerTableBatch struct {
 	t   *freezerTable
 	buf bytes.Buffer
 	sb  *BufferedSnapWriter
 
 	firstIdx uint64
 	count    uint64
 	sizes    []uint32
+	total    int
```

Comment:
Is it worth tracking `total`? I would have thought that `batch.buf.Len()` is fast enough

Reply:
Yes, I will change it back